make_webshots.py

```diff
@@ -165,10 +169,26 @@ def reset_driver(self):
         finally:
             self.set_driver()
 
-    def wait_no_progressbar(self, cls):
-        WebDriverWait(self.driver, 300, poll_frequency=0.1).until(
+    def wait_no_progressbar(self, cls, wait_appear=0):
+        if wait_appear:
+            # this is a dirty solution to the fact that now progress bar might not
+            # even appear for awhile, or at all (e.g. for listing an empty dandiset)
+            try:
+                log.debug("Wait for progress bar %s to appear", cls)
+                t0 = time.time()
+                out = WebDriverWait(self.driver, wait_appear, poll_frequency=0.05).until(
+                    EC.visibility_of_element_located((By.CLASS_NAME, cls))
+                    )
+                log.debug(" %s appeared after %fs", cls, time.time() - t0)
+            except TimeoutException as e:
+                log.debug(" %s failed to appear within %fs, continuing", cls, wait_appear)
+                return False  # no need to wait -- it did not come
```

Comment:
What's the point of returning a value?  The return value appears to be unused.

Reply:
Yeah, no point ATM (thought to make outside logic to subtract that time if false, didn't go for it) but also no harm